indico/legacy/pdfinterface/base.py

```diff
@@ -503,13 +503,14 @@ def _doNothing(canvas, doc):
 
 class DocTemplateWithTOC(SimpleDocTemplate):
 
-    def __init__(self, indexedFlowable, filename, firstPageNumber = 1, **kw ):
+    def __init__(self, indexedFlowable, filename, firstPageNumber=1, **kw):
         """toc is the TableOfContents object
         indexedFlowale is a dictionnary with flowables as key and a dictionnary as value.
             the sub-dictionnary have two key:
                 text: the text which will br print in the table
                 level: the level of the entry( modifying the indentation and the police
         """
+        self.include_table_of_contents = kw.get('include_table_of_contents', False)
```

Comment:
Why not add this as a kwarg of the constructor?

Reply:
don't mind changing it

Reply:
changed